libexec/cli/bootstrap.help

```diff
@@ -101,6 +104,22 @@ DEFFILE SECTION EXAMPLES:
         export FOO=bar
 
 
+DEFFILE SCI-F SECTION EXAMPLES:
+
+    %appinstall app1
+        echo "These are steps to install an app using the SCI-F format"
+
+    %appenv app1
+        APP1VAR=app1value
+        export APP1VAR
+
+    %apphelp app1
+        This is a help doc for running app1
+
+    %apprun app1
+        echo "this is a runscript for app1"
+
+
 EXAMPLES:
 
     $ singularity create /tmp/Debian.img
```

Comment:
should we remove the create from examples? Are we doing that anymore?

Reply:
We are still doing create before a bootstrap.  At least that is the way I set it up with the `--exists` option.  This may change when I remove the option.  But anyway, the idea is that many folks use scripts to `--create` and `--bootstrap` containers and we want to give them warnings before we jerk the rug out from under them.